modules/kafka/kafka_helpers_test.go

```diff
@@ -9,101 +9,171 @@ import (
+	}
+
+	t.Run("voters/localhost", func(t *testing.T) {
+		testConfigureControllerQuorumVotersFn(t, &testcontainers.GenericContainerRequest{
+			ContainerRequest: testcontainers.ContainerRequest{
+				Env: map[string]string{},
 			},
-			expectedVoters: "1@localhost:9094",
-		},
-		{
-			name: "voters on first network alias of the first network",
-			req: &testcontainers.GenericContainerRequest{
-				ContainerRequest: testcontainers.ContainerRequest{
-					Env:      map[string]string{},
-					Networks: []string{"foo", "bar", "baaz"},
-					NetworkAliases: map[string][]string{
-						"foo":  {"foo0", "foo1", "foo2", "foo3"},
-						"bar":  {"bar0", "bar1", "bar2", "bar3"},
-						"baaz": {"baaz0", "baaz1", "baaz2", "baaz3"},
-					},
+		}, "1@localhost:9094")
+	})
+
+	t.Run("voters/first-network-alias/first-network", func(t *testing.T) {
+		testConfigureControllerQuorumVotersFn(t, &testcontainers.GenericContainerRequest{
+			ContainerRequest: testcontainers.ContainerRequest{
+				Env:      map[string]string{},
+				Networks: []string{"foo", "bar", "baaz"},
+				NetworkAliases: map[string][]string{
+					"foo":  {"foo0", "foo1", "foo2", "foo3"},
+					"bar":  {"bar0", "bar1", "bar2", "bar3"},
+					"baaz": {"baaz0", "baaz1", "baaz2", "baaz3"},
 				},
 			},
-			expectedVoters: "1@foo0:9094",
-		},
-		{
-			name: "voters on localhost if alias but no networks",
-			req: &testcontainers.GenericContainerRequest{
-				ContainerRequest: testcontainers.ContainerRequest{
-					NetworkAliases: map[string][]string{
-						"foo":  {"foo0", "foo1", "foo2", "foo3"},
-						"bar":  {"bar0", "bar1", "bar2", "bar3"},
-						"baaz": {"baaz0", "baaz1", "baaz2", "baaz3"},
-					},
+		}, "1@foo0:9094")
+	})
+
+	t.Run("voters/localhost/alias-no-networks", func(t *testing.T) {
+		testConfigureControllerQuorumVotersFn(t, &testcontainers.GenericContainerRequest{
+			ContainerRequest: testcontainers.ContainerRequest{
+				NetworkAliases: map[string][]string{
+					"foo":  {"foo0", "foo1", "foo2", "foo3"},
+					"bar":  {"bar0", "bar1", "bar2", "bar3"},
+					"baaz": {"baaz0", "baaz1", "baaz2", "baaz3"},
 				},
 			},
-			expectedVoters: "1@localhost:9094",
-		},
-	}
+		}, "1@localhost:9094")
+	})
+}
 
-	for _, test := range tests {
-		t.Run(test.name, func(t *testing.T) {
-			configureControllerQuorumVoters(test.req)
+func TestValidateKRaftVersion(t *testing.T) {
+	validateKRaftVersionFn := func(t *testing.T, image string, wantErr bool) {
+		t.Helper()
+
+		err := validateKRaftVersion(image)
 
-			require.Equalf(t, test.expectedVoters, test.req.Env["KAFKA_CONTROLLER_QUORUM_VOTERS"], "expected KAFKA_CONTROLLER_QUORUM_VOTERS to be %s, got %s", test.expectedVoters, test.req.Env["KAFKA_CONTROLLER_QUORUM_VOTERS"])
-		})
+		if wantErr {
+			require.Error(t, err)
+		} else {
+			require.NoError(t, err)
+		}
 	}
+
+	t.Run("official/valid-version", func(t *testing.T) {
+		validateKRaftVersionFn(t, "confluentinc/confluent-local:7.5.0", false)
+	})
```

Comment:
suggestion: use direct to make it clearer, removing the function above and calling the methods directly.
```suggestion
	t.Run("official/valid-version", func(t *testing.T) {
		err := validateKRaftVersion("confluentinc/confluent-local:7.5.0")
		require.NoError(t, err)
	})
```